pkg/kubectl/cmd/drain_test.go

```diff
@@ -32,12 +32,13 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-
+	"k8s.io/api/core/v1"
```

Comment:
@shiywang I am sorry, I lost the part where you defined api.Node as internal type. Can you please elaborate?

Also, I was under the impression that these unit tests are supposed to test the exact the changes that we made in code. For example, if you look at taints etc, they are testing against v1.